backend/src/services/smtp/smtp.ts

```diff
@@ -193,7 +193,7 @@ async function sendEmail(email: Mail.Options) {
     log.info({ messageId: info.messageId }, 'Email sent')
   } catch (err) {
     const content = { to: email.to, subject: email.subject, text: email.text }
-    log.warn(content, `Unable to send email`)
+    log.warn({ content, err }, `Unable to send email`)
```

Comment:
Super minor, but as this will show in the logs can we call this `error` instead of `err`?